lua/toggleterm/terminal.lua

```diff
@@ -9,6 +9,9 @@ local api = vim.api
 local fmt = string.format
 local fn = vim.fn
 
+local command_sep = vim.fn.has("win32") and "&" or ";"
```

Comment:
@l-kershaw I believe this is the issue. `fn.has` returns a number so will always be truthy. I'm guessing you are on windows which is why it's working and I'm on linux where it's not